etc/requirements.txt

```diff
@@ -62,3 +62,6 @@ sortedcontainers==1.4.4
 intervaltree==2.1.0
 
 cachetools==1.1.5
+
+# For financial risk calculations
+qrisk>=0.1.4
```

Comment:
We should peg the version to exactly 0.1.4.  Currently we use this like a constraints file for our environments, and setup.py does some replacement to make pip installing zipline require `>=` the versions specified here.